src/sass/modules/_m-schemaform.scss

```diff
@@ -508,11 +508,11 @@ legend.schemaform-label.schemaform-file-label {
   margin-top: 0;
 }
 
-.react-autosuggest__container {
+.autosuggest-container {
```

Comment:
It looks like there might be a styling issue on mobile:
<img width="637" alt="screen shot 2018-01-04 at 4 12 36 pm" src="https://user-images.githubusercontent.com/16051172/34590233-37996c78-f16a-11e7-9fdb-43a9e87ac0ed.png">